css/css-scroll-anchoring/fragment-scrolling-anchors.html

```diff
@@ -48,7 +48,13 @@
     var ch = document.getElementById('changer');
     ch.style.height = 100;
 
+    await (new Promise((resolve) => window.addEventListener('scroll', resolve)));
```

Comment:
```suggestion
    await new Promise(resolve => addEventListener('scroll', resolve, {once: true}));
```

Do we want `{once: true}` here?